metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro/src/main/config/zookeeper/enrichments/bro.json

```diff
@@ -0,0 +1,17 @@
+{
+    "enrichment" : {
```

Comment:
This is seen more in METRON-942, because we actually install 3rd party extensions.  Which includes installing default configurations with the extension configuration pushed into zookeeper.

What I would like to get to is after installing an extension, we can create multiple instances of that extension, by cloning and editing the default configurations that are installed with it.

I also track which parser instances come from which extensions,.